### ISSUE TITLE
Problems using the prebuild image for Raspberry Pi 3

### ISSUE THREAD

**Issue by gerdb** [cardonabits/haxo-rs#15]
I've problems using the prebuild image. I have an RPI 3B+ and I've used the image for RPI3/4
Tried it with different micro SD cards (16GB and 32GB). Wrote the image with Ubuntu. The Raspberry hangs during booting:

![initramfs](https://github.com/cardonabits/haxo-rs/assets/883695/30d7425b-0658-44f0-bb22-22c2333ee495)

**Comment by jcard0na** [comment]
I don't have a Raspberry Pi 3B+ in hand.  I'll re-test the image with a RPI4.  This is what I'll try:
  1. Download image: https://bit.ly/3Qq9h5d
  2. Image with Raspberry Pi Imager
  3. Test

As soon as I have run this I'll report back.  Does the RPI3 boot *without* the Haxophone HAT attached and that image?

**Comment by gerdb** [comment]
I've used the Ubuntu disks tool to write the image to the sd-card. I'll try the Raspberry PI imager instead.

**Comment by jcard0na** [comment]
Hi @gerdb,

No need to try the RPI Imager, I was able to repro here.
Fortunately I could find an old RPI 3 to test it out and I got the same problem:
![IMG_3245 2](https://github.com/cardonabits/haxo-rs/assets/676181/10ed4ab0-2351-4e16-8976-389a165388f1)

The image works on Rasbperry Pi 4:
![IMG_3244](https://github.com/cardonabits/haxo-rs/assets/676181/41b2b496-765f-4986-983b-e5afef81398f)

I had assumed that the two images were compatible but they are not.  For the moment I will update the documentation and add to my queue creating an image for Raspberry Pi 3.

Sorry for the wasted time, and thank you very much for reporting.
